### ISSUE TITLE
With this project it will be possible to play "offline with bots" in the future?

### ISSUE THREAD

**Issue by MrBinOff** [phikill/RE_PUBG_LITE#2]

**Comment by phikill** [comment]
Yes, I am decompiling and remaking many parts of the classes so that they are very close to the game's original source code structure. To make the game work it will be necessary to create a local server, I am redoing the same steps so that it has a similar logic to how the game worked when it was alive.
